src/main/java/me/tomsdevsn/hetznercloud/HetznerCloudAPI.java

```diff
@@ -1905,7 +2006,18 @@ public ActionResponse changeProtectionOfLoadBalancer(long id, boolean delete) {
     public String convertToISO8601(Date date) {
         DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
         dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
-        String formatted = dateFormat.format(date);
-        return formatted;
+        return dateFormat.format(date);
+    }
+
+    private String getPaginationParameters(PaginationParameters pageParams) {
+        return pageParams != null ? getPageParameterIfNotNull(pageParams.getPage()) + getPerPageParameterIfNotNull(pageParams.getPerPage()) : "";
+    }
+
+    private String getPageParameterIfNotNull(Integer page) {
+        return page != null ? "?page=" + page :  "";
+    }
+
+    private String getPerPageParameterIfNotNull(Integer perPage) {
+        return perPage != null ? "?per_page=" + perPage : "";
```

Comment:
Since you only use this method in `HetznerCloudAPI#getPaginationParameters`, I would recommend you to change `?per_page=` to `&per_page=`, otherwise it will not work.

If you want to reuse this method, you need to replace `?` with `&` in `HetznerCloudAPI#getPaginationParameters`